Discussion/Discussion/Presentation/Posts/PostsView.swift

```diff
@@ -242,13 +230,70 @@ public struct PostsView: View {
                 Theme.Colors.background
                     .ignoresSafeArea()
             )
-            // MARK: - Action Sheet
-            .actionSheet(isPresented: $showingAlert, content: {
-                ActionSheet(title: Text(DiscussionLocalization.Posts.Alert.makeSelection), buttons: viewModel.filterButtons)
-            })
         }
     }
     
+    private var filterButton: some View {
+        Button(
+            action: {
+                showFilterSheet = true
+            }, 
```

Comment:
Here is extra white space and giving the warning `Trailing Whitespace Violation: Lines should not have trailing whitespace (trailing_whitespace)`.

Reply:
done